public/application.js

```diff
@@ -1,9 +1,8 @@
 $(document).ready(() => {
-  humanVersusHuman();
-  humanVersusComputer();
+  play();
 });
 
-function humanVersusComputer() {
+function play() {
   $("td").click(function(e) {
```

Comment:
use the es6 anonymous function syntax instead, aka `(e) => {}`

Reply:
:+1: